contrib/google.golang.org/grpc/server.go

```diff
@@ -6,6 +6,11 @@
 package grpc
 
 import (
+	"google.golang.org/grpc/metadata"
```

Comment:
The usual grouping we have is:
1. Local imports
2. External imports

You would then have:
```
import (
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"

	"github.com/golang/protobuf/jsonpb"
	"github.com/golang/protobuf/proto"
	context "golang.org/x/net/context"
	"google.golang.org/grpc"
	"google.golang.org/grpc/metadata"
)
```